src/components/side-panel/side-panel.vue

```diff
@@ -93,7 +93,7 @@ watch(infoOpen, infoOpen => {
   </div>
 
   <!-- Info panel -->
-  <div v-if="infoOpen" class="w-full md:w-80 bg-secondary z-10">
+  <div v-show="infoOpen" class="w-full md:w-80 bg-secondary z-10">
```

Comment:
same as first comment above, any reason why you are hiding the panel instead of remove it from the dom like the others?

Reply:
streetview needs to be persistent